### DISCUSSION TITLE
Where is this? "Backend > Dashboard > Updates"

### DISCUSSION THREAD

**Discussion by antforager** [bootscore/bootscore#842] [Q&A] [answered]
This is a screenshot from this page:  https://bootscore.me/blog/bootscore-6-0-1/
![image](https://github.com/user-attachments/assets/0b6a8ad7-e951-4c5e-9b56-cb3e62cb1cf6)

I have been looking for this in the Wordpress dashboard and can't find it.

**Comment by crftwrk** [comment, marked as answer]
Since v6 stable, like all plugins, the theme has the [PUC](https://github.com/YahnisElsts/plugin-update-checker) installed which checks by itself once a day if an update is available. Some servers are a bit slow here and you can force checking for updates in WP Dashboard > Updates. 

<img width="426" alt="screenshot" src="https://github.com/user-attachments/assets/4ea1506d-446d-4aed-a24c-33b41a124915">
